apps/frontend/src/app/5_pages/ProposalPage/components/ProposalInfo/ProposalInfo.module.css

```
@@ -0,0 +1,7 @@
+.markdown {
+  @apply w-full text-xs text-gray-30 break-words sm:break-words;
```

```suggestion
  @apply w-full text-xs text-gray-30;
```
I think this not needed here